src/scopes/activityReport/reportText.js

```diff
@@ -1,170 +1,79 @@
+  return selectDistinctActivityReports(
+    '',
+    '',
+    `${a} LOWER(STRING_AGG(concat_ws(CHR(10), "ActivityReports"."context", "ActivityReports"."additionalNotes"), CHR(10)))`,
+  );
 };
 
 export function withReportText(searchText) {
-  const search = [`%${searchText}%`];
+  const search = [`%${searchText.map((st) => st.toLowerCase())}%`];
 
   return {
     [Op.or]: [
-      filterAssociation(nextStepsPosNeg(true), search, false, 'ILIKE'),
-      filterAssociation(argsPosNeg(true), search, false, 'ILIKE'),
-      filterAssociation(objectiveTitlePosNeg(true), search, false, 'ILIKE'),
-      filterAssociation(objectiveTtaProvidedPosNeg(true), search, false, 'ILIKE'),
-      filterAssociation(activityReportResourcePosNeg(true), search, false, 'ILIKE'),
-      filterAssociation(activityReportGoalResourcePosNeg(true), search, false, 'ILIKE'),
-      filterAssociation(activityReportObjectiveResourcePosNeg(true), search, false, 'ILIKE'),
-      filterAssociation(nextStepsResourcePosNeg(true), search, false, 'ILIKE'),
-      filterAssociation(activityReportContextPosNeg(true), search, false, 'ILIKE'),
-      filterAssociation(additionalNotesPosNeg(true), search, false, 'ILIKE'),
+      filterAssociation(nextStepsPosNeg(true), search, false, 'LIKE'),
+      filterAssociation(argsPosNeg(true), search, false, 'LIKE'),
+      filterAssociation(objectiveTitleAndTtaProvidedPosNeg(true), search, false, 'LIKE'),
+      filterAssociation(activityReportContextandAdditionalNotesPosNeg(true), search, false, 'LIKE'),
     ],
   };
 }
 
 export function withoutReportText(searchText) {
-  const search = [`%${searchText}%`];
+  const search = [`%${searchText.map((st) => st.toLowerCase())}%`];
 
   return {
     [Op.and]: [
-      filterAssociation(nextStepsPosNeg(false), search, false, 'NOT ILIKE'),
-      filterAssociation(argsPosNeg(false), search, false, 'NOT ILIKE'),
-      filterAssociation(objectiveTitlePosNeg(false), search, false, 'NOT ILIKE'),
-      filterAssociation(objectiveTtaProvidedPosNeg(false), search, false, 'NOT ILIKE'),
-      filterAssociation(activityReportResourcePosNeg(false), search, false, 'NOT ILIKE'),
-      filterAssociation(activityReportGoalResourcePosNeg(false), search, false, 'NOT ILIKE'),
-      filterAssociation(activityReportObjectiveResourcePosNeg(false), search, false, 'NOT ILIKE'),
-      filterAssociation(nextStepsResourcePosNeg(false), search, false, 'NOT ILIKE'),
-      filterAssociation(activityReportContextPosNeg(false), search, false, 'NOT ILIKE'),
-      filterAssociation(additionalNotesPosNeg(false), search, false, 'NOT ILIKE'),
+      filterAssociation(nextStepsPosNeg(false), search, false, 'NOT LIKE'),
+      filterAssociation(argsPosNeg(false), search, false, 'NOT LIKE'),
+      filterAssociation(objectiveTitleAndTtaProvidedPosNeg(false), search, false, 'NOT LIKE'),
+      filterAssociation(activityReportContextandAdditionalNotesPosNeg(false), search, false, 'NOT LIKE'),
```

Comment:
Nothing that has to be changed but might want to sub "PosNeg" language with "IncludeExclude".